package.json

```diff
@@ -895,6 +895,10 @@
     "dev:vsix": "npm run clean && npm install && npm run fetch:wasm && npm run fetch:docs && npm run fetch:spdx-licenses && npm run compile && npm run package",
     "dev:npm": "npm run clean && npm install && npm run fetch:wasm && npm run fetch:docs && npm run fetch:spdx-licenses && cd server && npm pack"
   },
+  "//": "TODO Bump eslint-plugin-header when rules is compatible with eslint 9",
+  "//": "See https://github.com/Stuk/eslint-plugin-header/issues/57",
+  "//": "TODO Bump vsce when symlink regression has been fixed",
+  "//": "See https://github.com/microsoft/vscode-vsce/pull/1053",
```

Comment:
This is not a valid syntax in JSON files because there are duplicated keys. Can't we just create a ticket in the backlog for it?

Reply:
This is valid in package.json which has this feature specifically for comments. I think we'll more easily catch up if they are present in the code.

Reply:
npm strips the '//' entries before parsing the JSON: https://stackoverflow.com/questions/14221579/how-do-i-add-comments-to-package-json-for-npm-install

Reply:
oh cool, that's interesting to know.